src/verify_expr.ml

```diff
@@ -420,28 +422,28 @@ module VerifyExpr(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
   
   let interfmap1 =
     List.map
-      begin fun (ifn, (l, fieldmap, specs, preds, interfs, pn, ilist)) ->
+      begin fun (ifn, (l, fieldmap, specs, preds, interfs, pn, ilist, tparams)) ->
         let mmap =
-        let rec iter mmap meth_specs =
+        let rec iter mmap erased_mmap meth_specs =
           match meth_specs with
             [] -> List.rev mmap
           | Meth (lm, gh, rt, n, ps, co, body, binding, _, _)::meths ->
-            if body <> None then static_error lm "Interface method cannot have body" None;
-            if binding = Static then static_error lm "Interface method cannot be static" None;

```

Comment:
Waarom heb je dit verwijderd? Maak ongedaan. Als je ondersteuning voor interface methods met bodies of statische interface methods wilt toevoegen, moet dat in een aparte PR.